packages/sdk/Dockerfile

```diff
@@ -61,6 +71,8 @@ COPY entrypoint.sh /usr/local/bin/
 COPY retar /usr/local/bin/
 COPY --from=genext2fs /usr/local/bin/genext2fs /usr/bin/
 COPY --from=su-exec /usr/local/src/su-exec /usr/local/bin/
+COPY --from=undocker /undocker/undocker /usr/local/bin/
+RUN mkdir -p /tmp/.sunodo && chmod 1777 /tmp/.sunodo
```

Comment:
Do we need to create this directory here?

Reply:
Yes, the build failed without it.